styles.css

```diff
@@ -1,219 +1,10 @@
+@import url("base.css");
+@import url("utility.css");
+@import url("buttons.css");
+@import url(headlines-and-captions.css);
+
 @import url("https://fonts.googleapis.com/css2?family=Bebas+Neue&display=swap");
 @import url("https://fonts.googleapis.com/css2?family=Arvo&family=Bebas+Neue&family=Roboto+Slab:wght@200&display=swap");
 * {
   box-sizing: border-box;
 }
```

Comment:
I would recommend adding the universal selector to your `base.css` file. The `styles.css` should only have the imports listed :)